Assignment02/step-by-step-java.md

```diff
@@ -237,6 +237,8 @@ Now you'll change the code to use the Dapr-provided `DaprClient` to call the Veh
 
    > Also note that the `invokeMethod` method of the `DaprClient` returns a `Mono`. Because we don't want to get distracted by all the possibilities that reactive progamming brings, we immediately invoke `block` to await the result. In a real-world scenario, it would make more sense to propagate that Mono through our application to make it more reactive.
 
+   > At this point, you could consider removing the `vehicle-information.address` entry from the `application.yml` file. The Dapr SDK for Java does not need it.
```

Comment:
Should we make this more explicit and tell them to remove it? For the good cause? :)

Reply:
Yep, done.